src/ts/components/account/ImportJson.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import t from '../../services/i18n'
+import { importAccountFromJson, decodeAddress } from '../../services/keyring-vault-proxy'
+import { RouteComponentProps, withRouter } from 'react-router'
+import { KeyringPair$Json } from '@polkadot/keyring/types'
+import { Message } from 'semantic-ui-react'
+import Dropzone from 'react-dropzone'
+import { isObject, u8aToString, isHex } from '@polkadot/util'
+import { Section, Button } from '../basic-components'
+
+interface ImportJsonProp extends RouteComponentProps {
+  history: any
```

Comment:
we don't need define history any more. It is included in RouteComponentProps already

Reply:
ok, pushed